src/app/main/component/events/components/create-edit-events/create-edit-events.component.html

```diff
@@ -3,37 +3,36 @@
   <a [routerLink]="['/events']">
     <div class="button-arrow">
       <img [src]="images.arrowLeft" aria-hidden="true" alt="arrow" class="button-arrow-img" />
-      <div class="edit-news">Back</div>
+      <div class="edit-news">{{ 'create-event.back' | translate }}</div>
     </div>
   </a>
   <div class="event-header">
-    <h3 *ngIf="!editMode">Create Event</h3>
-    <h3 *ngIf="editMode">Edit Event</h3>
+    <h3 *ngIf="!editMode">{{ 'create-event.title' | translate }}</h3>
+    <h3 *ngIf="editMode">{{ 'create-event.edit' | translate }}</h3>
     <p>
-      Please provide as much details as you can - place and time of event, the goal of gathering etc. You can come back and update event
-      anytime after publishing
+      {{ 'create-event.description' | translate }}
     </p>
   </div>
   <div class="event-main">
     <mat-card class="event-main-block left">
       <div class="main-title">
         <div class="title-input">
           <div class="title-wrapper">
-            <mat-label>Enter name for Event</mat-label>
-            <mat-label>{{ eventFormGroup.get('titleForm').value.length }}/70</mat-label>
+            <mat-label>{{ 'create-event.event-name' | translate }}</mat-label>
+            <mat-label>{{ eventFormGroup.get('titleForm').value.length }}{{ 'create-event.event-name-length' | translate }}</mat-label>
```

Comment:
Don't need to add translation for /70 - replace {{ 'create-event.event-name-length' | translate }} with /70